src/pybind/rbd/setup.py

```diff
@@ -20,9 +20,17 @@ def get_version():
     try:
         for line in open(os.path.join(dir, "ceph_ver.h")):
             if "CEPH_GIT_NICE_VER" in line:
-                return line.split()[2][1:-1]
+                l=line.split()[2].strip('"')
+                if len(l) > 1 and l[0] == 'v': l = l[1:]
+                a=l.split("-",2)
+                if len(a) > 2:
+                    return "{}.post{}+{}".format( a[0], a[1], a[2] )
```

Comment:
i think we can just use git version, i.e. x.y.z for the extension's version. and iiuc, we are not using the post version,

quote from https://www.python.org/dev/peps/pep-0440/#post-releases
> Some projects use post-releases to address minor errors in a final release that do not affect the distributed software (for example, correcting an error in the release notes).

 so X.postY.Z does not apply to our project, right?